web-ui/yarn.lock

```diff
@@ -6477,7 +6477,7 @@ debug@=3.1.0:
   dependencies:
     ms "2.0.0"
 
-debug@^3.1.1, debug@^3.2.5:
+debug@^3.0.0, debug@^3.1.1, debug@^3.2.5:
```

Comment:
why is 3.0.0 version added ?

Reply:
not sure, I think it got added after build.